lib/beaker-hostgenerator/hypervisor.rb

```diff
@@ -84,7 +84,7 @@ def global_config()
       #
       # @param [Integer] bhg_version The version of OS info to use when building
       #                              up the node definition.
-      def generate_node(node_info, base_config, bhg_version)
```

Comment:
why define parameters when you plan to never use them? (_ syntax)

Reply:
there are different implementations of this class, for example for hcloud, vagrant, docker. They all have to provide the method `generate_node` with the same signature. This is just the dummy/base implementation. I think it has to have the same signature as well.